templates/Uwp/Services/SqlServerDataService.Prism/.template.config/template.json

```diff
@@ -14,7 +14,7 @@
     "wts.platform": "Uwp",
     "wts.group": "Data",
     "wts.version": "1.0.0",
-    "wts.displayOrder": "1",
+    "wts.displayOrder": "s",
```

Comment:
Should this be a 2?

Reply:
Yes. Will also update the validator tests as they should have picked that up.

Reply:
There is a test that should have caught this. Now to investigate why it didn't...